cmd/minikube/cmd/start_flags.go

```diff
@@ -220,7 +220,7 @@ func generateClusterConfig(cmd *cobra.Command, existing *config.ClusterConfig, k
 			glog.Warningf("Unable to query memory limits: %v", err)
 		}
 
-		mem := suggestMemoryAllocation(sysLimit, containerLimit)
+		mem := suggestMemoryAllocation(sysLimit, containerLimit, viper.GetInt(nodes))
```

Comment:
does this getInt(Nodes) return 2 if u add a node to an existing start?

Reply:
nope, "nodes" is only a request parameter for `minikube start`